package.json

```diff
@@ -161,7 +161,7 @@
   },
   "packageManager": "pnpm@10.10.0",
   "engines": {
-    "node": ">=18.18.0"
+    "node": ">=20.9.0"
```

Comment:
<img width="1172" alt="image" src="https://github.com/user-attachments/assets/179ae9e1-7b26-45de-9224-6f7e73a998de" />

I think we gotta target ES2023 with 20.11.0?